### ISSUE TITLE
What Features are Used in the Geometric Pre-training Tasks in GeoLayoutLM?

### ISSUE THREAD

**Issue by minhoooo1** [AlibabaResearch/AdvancedLiterateMachinery#68]
I appreciate the work done with GeoLayoutLM, but I find that the explanation about the geometric pre-training tasks in the paper is not very clear. The paper mentions "The input of these tasks are text-segments feature {Bi} which can be either of the five features:{Hi},{Mvi},{Fvi},{Mt,b(i)},{Fi,b,(i)}, and then there pre-training tasks such as P = Softmax(Linear[Bi,Bj])) (Direction and Distance Modeling) are set up. However, it's not specified which of the five features is the Bi used in this task. Could you please provide more clarity on this matter?

**Comment by ccx1997** [comment]
The geometric pre-training objective is exerted to the 5 features respectively. The final objective is the sum of them, plus a mvlm loss.

**Comment by minhoooo1** [comment]
Thank you very much for the prompt reply

**Comment by minhoooo1** [comment]
By the way, do the features of the mask tokens not participate in these geometric pre-training tasks?

**Comment by ccx1997** [comment]
> By the way, do the features of the mask tokens not participate in these geometric pre-training tasks?

They still probably participate in geometric pre-training tasks, since their coordinates are not masked.

**Comment by minhoooo1** [comment]
Thank you very much for the prompt reply
